xonsh/procs/jobs.py

```diff
@@ -239,10 +242,7 @@ def wait_for_active_job(last_task=None, backgrounded=False, return_error=False):
             except subprocess.TimeoutExpired:
                 pass
             except KeyboardInterrupt:
-                try:
-                    _kill(active_task)
-                except subprocess.CalledProcessError:
-                    pass  # ignore error if process closed before we got here
+                _ctrl_c(active_task)
```

Comment:
I actually don't think you need to send *anything*. Both the Python REPL and NeoVim (installed by winget) do their CTRL-C handling just fine on their own, tested both in Windows Terminal and in Command Prompt (cmd.exe).

When you do this `_ctrl_c()` here, processes get a duplicate CTRL-C in short succession. In case of Python 3.13 from the Windows Store it sends it into an infinite loop of KeyboardInterrupts.

Just saying `pass` here works fine for me.

Reply:
Validated locally, thanks! 👍